user-info-provider/README.md

```diff
@@ -135,3 +135,57 @@ This mapper produces the following standardized metadata JSON-structure
 }
 ```
 
+## Configuration based on role
+There is now a possibility to hide the person tag in the claim based on the role attribute which the user has. If this
+needs to be used please add the professional roles which need to be checked in the mapper and select the negate output
+accordingly.
+
+<img src="../docs/userinfo/Vidis-role-based-info-mapper.png" width="70%"/>
+
+# Role based info Mapper
+
+Service Provider Mapper similar to the user property mapper provided by Keycloak, which reads provided property from the
+user attribute based on the role which is assigned to the user and adds them to the token. Role is checked in the following
+user attributes : rolle, person.kontext.rolle, person.kontext.<number>.rolle.
+
+## Input
+This mapper uses the following standardized user attributes for roles as an input:
+[user-info-api](./../user-info-api/README.md)
+
+## Configuration
+If a Service Provider supports processing users metadata, a new mapper has to be configured for it.
+1. Go to "Clients" and select the client to configure
+2. Go to Tab "Mappers" and click on Button "create" adding a new mapper
```

Comment:
to be honest: I think to know how to add a mapper should be prerequisite to even being allowed to configure this mapper...